packages/opencensus-exporter-stackdriver/README.md

```diff
@@ -55,6 +55,42 @@ or
 tracing.registerExporter(exporter).start();
 ```
 
+## Monitoring Exporter
+
+To use Stackdriver as your exporter, make sure you have enabled [Stackdriver Monitoring](https://cloud.google.com/monitoring/docs/quickstart) on Google Cloud Platform. Enable your [Application Default Credentials](https://cloud.google.com/docs/authentication/getting-started) for authentication with:
+
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS=path/to/your/credential.json
+```
+
+Instance the exporter on your application passing your Project ID and register your exporter on Stats. For javascript:
```

Comment:
Nit: Capitalize `javascript` to JavaScript.

Technically, the code written for TS is also JavaScript. Maybe it should be written in a way (e.g. with types) that suggests that types are usable.

Reply:
So, I kept JS and TS separated because we did it this way for Trace, but I don't think we should keep it. As you said, TS is also JS.

I made a few changes to remove this separation, but let me know if we should keep them as they were.

Reply:
Well, TS is not JS in general... I was just pointing out that these TS code snippets happen not to have any details specific to TypeScript at all.

Having seen the JS snippets removed, I actually think that if we were to keep only one, it should be the JS snippets, as these are the code samples that can actually be run if one C+P's them into a `.js` file and runs it.